### ISSUE TITLE
issue when upgrading Android Support Libraries

### ISSUE THREAD

**Issue by hectorv** [appium/appium#4926]
Our Android devs upgraded the support libraries and now certain things stopped working, like the ability to find/interact with the menu (ExpandedMenuView). This happens while using Selendroid in both Appium v1.3.3 and v1.3.7.

I then noticed that this one-liner in our `build.gradle` fixes automation (it just downgrades the v7 appcompt library):
```diff
-    compile 'com.android.support:appcompat-v7:22.0.+'
+    compile 'com.android.support:appcompat-v7:21.0.+'
```

Thanks!

**Comment by Jonahss** [comment]
Thank you. @paymand does this make sense to you?

**Comment by paymand** [comment]
Does this also happen with Appium 1.4.0-beta which has Selendroid 0.15.0?

**Comment by hectorv** [comment]
This seems to be fixed in Appium v1.4.0
